packages/common/src/hooks/purchaseContent/useChallengeCooldownSchedule.ts

```diff
@@ -27,8 +30,7 @@ export const useChallengeCooldownSchedule = (
   const challenges = useSelector(getUndisbursedUserChallenges)
     .filter((c) => c.challenge_id === challengeId)
     .map((c) => ({ ...c, createdAtDate: dayjs.utc(c.created_at) }))
-  // Clamp "now" to beginning of day to avoid duplicate dates in cooldown items
-  const now = dayjs.utc().startOf('day')
+  const now = dayjs.utc()
```

Comment:
So, the original comment said this is supposed to be the start of the day, but it's been replaced with the current time?

Reply:
yep that was a bug - don't want to clamp here bc we want to filter for the challenges that are still in cooldown, so strictly less than 7 days difference from now.

tbh i'm not entirely certain we need to clamp later on either, will investigate.